### DISCUSSION TITLE
Suggestions on being able to scale the Video Conferencing.

### DISCUSSION THREAD

**Discussion by 1ay1** [igniterealtime/openfire-pade-plugin#282] [Q&A]
Is the current system able to handle 1000 users concurrently in different rooms.
Can I use Openfire clustering for this?

**Comment by gjaekel** [comment]
This depends on your understanding of "system": To my knowledge, the software isn't limited in principle. But you have to deal with the limits introduced by your hardware. And think about your SLA requirements.

There's a read big non-commerical plattform in Germany offered by the Freifunkt e.V. (https://meet.ffmuc.net). They don't use OpenFire but Prosody the products as preffered by Jitsi.org. In peak, the handle more than 5000 participants. They use about >=20 Containers with Video bridges because the leased hardware is limited to run about 100-200 participants on each JVB node. For the XMPP server, they use a HA setup.

On the other hand, currently I'm able to handle ~120 (and rising) participants with typical conference sizes for the larger ones up 30 (and rising) to without any problems. The container running all OpenFire don't show up a load bigger than 10. The JVB "stress" (using the default reference value) peaks up to 1.5 and the maximum outgoing bandwidth usage is about 60kbyte/s.

**Comment by 1ay1** [comment]
Can I try using Openfire Clustering plugin to try to achieve the HA?

**Comment by gjaekel** [comment]
I don't have any experience with HA-OpenFire. Said that, the configuration of the JVB2's needs to be expanded to build a cluster. This will require support by the Plugin to generate the config files.

**Comment by gjaekel** [comment]
To get an idea of the requirements, you may look at https://stats.ffmuc.net/ and switch to the _Meet Stats_ (e.g. https://stats.ffmuc.net/d/U6sKqPuZz/meet-stats?orgId=1&refresh=10m&from=now-7d&to=now)
